docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,130 @@
-# Configuring Zowe certificates 
+# Zowe certificate configuration overview
 
-As a system administrator, review this article to learn about the key concepts of Zowe certificates.
+As a system programmer, review this article to learn about the key concepts of Zowe certificates, and options for certificate configuration.
 
-Zowe uses a certificate to encrypt data for communication across secure sockets. The certificate used is specified in the `zowe.certificates` section of the `zowe.yaml`, and can either be a `PKCS12` certificate held in a USS keystore `.p12` file, or else a `JCERACKS` certificate held in a keyring.  
+Zowe provides the ability to [generate a certificate](./use-certificates.md) using the `zwe init certificate` command. Zowe can also be configured to [use an existing certificate](./import-certificates.md) provided by the user's security team in a z/OS customer shop.
 
-Zowe provides the ability to generate a certificate using the `zwe init certificate` command, and it can also be configured to use an existing certificate that might be provided by the security team in a z/OS customer shop.
- 
-## Trust store
+## What Zowe certificates are used for?
```

Comment:
This reads a little awkward. A better suggestion might be: What are Zowe certificates used for?